mavros_msgs/CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ add_message_files(
   OpticalFlowRad.msg
   OverrideRCIn.msg
   ParamValue.msg
+  ManualControls.msg
```

Comment:
I can switch to singular, but I was worried it could be interpreted (by a
ROS user) as an input (i.e.: to have manual control). If you'd prefer
singular, I can change it.

Regarding the buttons field. I did not include it since pixhawk seems to
send zero. Should I add it anyway?

On Fri, Aug 28, 2015 at 4:57 PM, Vladimir Ermakov <notifications@github.com>
wrote:

> In mavros_msgs/CMakeLists.txt
> <https://github.com/mavlink/mavros/pull/377#discussion_r38238028>:
>
> > @@ -17,6 +17,7 @@ add_message_files(
> >    OpticalFlowRad.msg
> >    OverrideRCIn.msg
> >    ParamValue.msg
> > +  ManualControls.msg
>
> Why plural form? Please name it ManualControl.msg like corresponding
> mavlink message.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mavlink/mavros/pull/377/files#r38238028>.
>